setup.cfg

```diff
@@ -14,21 +14,24 @@ zip_safe = False
 python_requires = >=3.7
 packages = find:
 install_requires =
-    astropy
+    astropy>=4.0
     glue-core>=0.15
     sunpy
     reproject>=0.7
     ndcube
     dask[array]
     sunraster
+    echo>=0.2
```

Comment:
What is echo?

Reply:
Yeah, we used to import it via `glue`: http://docs.glueviz.org/en/stable/_modules/glue/external/echo.html

Reply:
But I have been warned that this usage is deprecated and that I should import `echo` directly.